pkg/daemon/ceph/osd/remove.go

```diff
@@ -138,5 +138,23 @@ func removeOSD(clusterdContext *clusterd.Context, clusterInfo *client.ClusterInf
 		logger.Errorf("failed to remove CRUSH host %q. %v", hostName, err)
 	}
 
+	// The ceph health warning should be silenced by archiving the crash
+	crash, err := client.GetCrash(clusterdContext, clusterInfo)
```

Comment:
I'd still recommend splitting this out to a separate func.

Reply:
I added GetCrash

Reply:
Yes, that's good for the helper methods. Even this would be good to split out to a separate method (in the same remove.go) so that we keep the `removeOSD()` func shorter. Shorter functions are easier to maintain. Lines 85-125 above should really be split out to another func as well, but somebody else can take care of that in a separate PR.

Reply:
?

Reply:
done